network/src/peer/test.rs

```diff
@@ -191,7 +191,8 @@ fn peer_send_message() {
 
     let client = async move {
         peer_handle.send_message(send_msg, PROTOCOL).await.unwrap();
-        ManuallyDrop::new(peer_handle);
+        //ManuallyDrop::new(peer_handle);
+        forget(peer_handle);
```

Comment:
This wasn't a lint, but a compiler warning:

unused return value of `ManuallyDrop::<T>::new` that must be used
`#[warn(unused_must_use)]` on by default
if you don't need the wrapper, you can use `mem::forget` insteadrustc(unused_must_use)